flash/data/__init__.py

```diff
@@ -1 +0,0 @@
-from flash.data.data_utils import labels_from_categorical_csv
```

Comment:
where is this going now ?

Reply:
Could import it directly `from flash.data.data_utils import labels_from_categorical_csv`.